cluster_management/src/main/java/com/pinterest/rocksplicator/ConfigGenerator.java

```diff
@@ -237,4 +242,27 @@ private boolean updateDisabledHosts() {
     disabledHosts = latestDisabledInstances;
     return true;
   }
+
+  /**
+   * filter out resources with "Task" state model (ie. workflows and jobs);
+   * only keep db resources from ideal states
+   */
+  public void filterOutTaskResources(List<String> resources) {
+    HelixAdmin admin = helixManager.getClusterManagmentTool();
+    for (Iterator<String> iter = resources.iterator(); iter.hasNext(); ) {
```

Comment:
nit: move Iterator<String> iter = resources.iterator() outside the loop and use while() instead of for()?